### ISSUE TITLE
Refreshing Kubernetes connection actions do not refresh primary resources

### ISSUE THREAD

**Issue by axel7083** [containers/podman-desktop#9679]
### Bug description

When you start your kind cluster, refresh the resources, pods are not visible. (never)

### Operating system

Fedora 40

### Installation Method

None

### Version

1.13.3

### Steps to reproduce

1. Start podman desktop
2. assert have existing context
3. Start kind cluster
4. Go to kubernetes tab, and click on `Refresh`
5. assert nodes of cluster are visible
6. Go to pods
7. assert pods not visible

https://github.com/user-attachments/assets/e8b4c269-5ffb-4854-993e-41d36214f921

### Relevant log output

N/a

**Comment by feloy** [comment]
Please note that the list of pods is displayed with its own pod watcher/lister, not with the informers in `contexts-manager` used for other resources (nodes, deployments, ...), and thus the refresh button does not have any effet on the pods list.

**Comment by odockal** [comment]
@axel7083 what pods exactly do you have in mind? There are pods on kind present after cluster creation, these are the ones you talk about or custom pods deployed on the kind, ie. from podman-desktop?

**Comment by axel7083** [comment]
> [@axel7083](https://github.com/axel7083) what pods exactly do you have in mind? 

I deployed a pod on kind, then stopped the cluster, stop podman-desktop, restart podman-desktop, wait 2 minutes, start the kind cluster, goto node, click on refresh, assert no pods listed but node listed

**Comment by odockal** [comment]
> > [@axel7083](https://github.com/axel7083) what pods exactly do you have in mind?
> 
> I deployed a pod on kind, then stopped the cluster, stop podman-desktop, restart podman-desktop, wait 2 minutes, start the kind cluster, goto node, click on refresh, assert no pods listed but node listed

Ah, so custom pod deployed. Ok, thanks for info!

**Comment by feloy** [comment]
setting on hold, depends on https://github.com/podman-desktop/podman-desktop/issues/10467

**Comment by feloy** [comment]
@deboer-tim do you think this is fixed by #10927 ?

**Comment by deboer-tim** [comment]
No - pods show fine on the new Kubernetes Pods page, but they don't show up on the regular Pods page because it still uses watchers. We either need to remove Kubernetes pods from the Pods page or do the technical debt to switch them over to informers. There's some disagreement on which we should do, hoping to close on it soon.

**Comment by feloy** [comment]
As I understand, Kubernetes pods have been removed from regular Pods page in https://github.com/podman-desktop/podman-desktop/issues/8819

I cannot find which PR removed the watchers, but I cannot see anymore the watchers which were present in v1.16.0: https://github.com/podman-desktop/podman-desktop/blob/v1.16.0/packages/main/src/plugin/kubernetes/kubernetes-client.ts#L337-L355

cc @deboer-tim

**Comment by deboer-tim** [comment]
Yes, the core problem here was fixed as part of the Pod move, and then watcher code removed. Podman Pods use the same events as containers; Kubernetes Pods now use the same informers as all other Kube resources (which will be improved by the experimental support). Closing.
